pages/admin/invite/userRequest.tsx

```diff
@@ -0,0 +1,111 @@
+import { useEffect, useState } from "react";
+import DeclineButton from "components/declineButton";
+import AcceptButton from "components/acceptButton";
+
+const UserRequestDashboardItem: React.FunctionComponent<UserRequest> = ({
+  name,
+  email,
+  phoneNumber,
+}) => {
+  return (
+    <div className="d-flex flex-row">
+      <div className="flex flex-row justify-between text-sm text-center">
+        <div className="d-flex flex-row">{name}</div>
+        <div className="d-flex flex-row">{email}</div>
+        <div className="d-flex flex-row">{phoneNumber}</div>
+        <div className="d-flex flex-column">
+          <DeclineButton />
+        </div>
+        <div className="flex flex-row justify-between">
+          <AcceptButton />
+        </div>
+      </div>
+      <hr className="border-unselected border-opacity-50" />
+    </div>
+  );
+};
+interface UserRequest {
+  name: string;
+  email: string;
+  image: string;
+  phoneNumber: string;
+}
+
+const UserDashboard: React.FunctionComponent = () => {
+  const [users, setUsers] = useState<UserRequest[]>();
+
+  const getUsers = async (pageNumber: number): Promise<void> => {
+    try {
+      const response = await fetch("http://localhost:3000/api/admin/users", {
+        method: "GET",
+        query: pageNumber,
```

Comment:
This generates a type error. please resolve it.

Argument of type '{ method: string; query: number; headers: { "content-type": string; }; redirect: "follow"; }' is not assignable to parameter of type 'RequestInit'.
  Object literal may only specify known properties, and 'query' does not exist in type 'RequestInit'.